src/qt/bitcoin.h

```diff
@@ -47,6 +48,8 @@ class BitcoinApplication: public QApplication
     void createOptionsModel(bool resetSettings);
     /// Initialize prune setting
     void InitPruneSetting(int64_t prune_MiB);
+    /** Set up translations */
+    void initTranslations(QTranslator &qtTranslatorBase, QTranslator &qtTranslator, QTranslator &translatorBase, QTranslator &translator);
```

Comment:
What about making it a static member function? I slightly like the idea of grouping it that way, but making it a normal member function is not needed)
(Sure, closing is also fine with me)